src/tribler-common/tribler_common/sentry_reporter/sentry_reporter.py

```diff
@@ -332,6 +333,14 @@ def get_actual_strategy():
         strategy = SentryReporter.thread_strategy.get()
         return strategy if strategy else SentryReporter.global_strategy
 
+    @staticmethod
+    def get_test_sentry_url():
+        return os.environ.get('TRIBLER_TEST_SENTRY_URL', None)
+
+    @staticmethod
+    def is_in_test_mode():
+        return SentryReporter.get_test_sentry_url() is not None
```

Comment:
**issue:** it may be convenient to treat empty value of an environment variable as a sign that it is not set

**suggestion**: we can treat the empty string the same way as a missing environment variable.

```suggestion
        return bool(SentryReporter.get_test_sentry_url())
```